### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Nivedita-coder**
Updating the chair and tech lead folder's readme.

### CODE REVIEW

**Review by mrbobbytables — COMMENTED** [review]
Thanks for getting this started, I left a few comments 👍 

One other potential item that would be worth listing under the CNCF resources is their SIG Contributor Strategy has a maintainer circle now:
https://github.com/cncf/sig-contributor-strategy/tree/master/maintainers-circle

**Comment by Nivedita-coder** [comment]
Hello @mrbobbytables !
I have done all the changes, please let me know if any more changes required.
Thank you.

**Comment by jdumars** [comment]
thanks for the contribution!

/lgtm

File: contributors/chairs-and-techleads/README.md [hunk 1]
```diff
@@ -1,36 +1,48 @@
...
+  [Governance Principles]
+  
+  ## Communication and feedback loops:
+  
+  - [Need to know email archive]
+  - [Monthly meeting]
+  
+  - Time:
+    * 2nd Tuesday of the Month 12:30pm PT [830pmUTC]
+    * 2nd Thursday of the Month 09:00am PT [5pmUTC]
```
Comment (mrbobbytables) [inline contributors/chairs-and-techleads/README.md:33]: qq - Are the 8:30 and 5pm meant to link to a timezone converter?

**Review by mrbobbytables — COMMENTED** [review]
Few more little things, but I'd consider them non-blocking :)

File: contributors/chairs-and-techleads/README.md (hunk 1 above)
Comment (Nivedita-coder) [reply contributors/chairs-and-techleads/README.md:32]: It is not linked to the timezone converter but if you will say I will add it.

Comment (mrbobbytables) [reply contributors/chairs-and-techleads/README.md:32]: kk, I saw the `[ ]` which usually surrounds a markdown link.^^;;;

Comment (Nivedita-coder) [reply contributors/chairs-and-techleads/README.md:32]: Okay, So what should I do , should I remove it !

Comment (mrbobbytables) [reply contributors/chairs-and-techleads/README.md:32]: Sorry, thought I approved this the other day.

**Pull request merged**